examples/karaoke.rs

```diff
@@ -62,24 +95,24 @@ fn wait_ctrlc() -> Result<(), Error> {
 }
 
 fn main() -> Result<(), Error> {
-    // Monoral microphone.
-    let input_channels = 1;
-    // Monoral speaker.
-    let output_channels = 1;
+    let args = Args::from_args();
+    let config = AppConfig::from_file_or_defaults(&args.config_file)?;
```

Comment:
The example code only works if `num_capture_channels==1` and `num_render_channels==1 or 2`. Let's assert it here.